indico/modules/events/contributions/templates/forms/contribution_field.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'forms/form_common_fields_first.html' %}
+
+{% block javascript %}
+    <script>
+        var $userEditable = $('#user_editable');
```

Comment:
Hardcoded IDs in widget JS are generally not a good idea. Also, have you checked the `HiddenUnless` validator? It automatically enables JS to hide the field if the condition is not met.

Reply:
But Hidden unless applies only when we want to hide a field, not an option of the field, right?

Reply:
ah it's just an option.. you're right then. anyway, still not a good idea to hardcode the ids. you should be able to get them dynamically via the field (`field.id`, `field.get_form().otherfield.id`)